python/taichi/core/util.py

```diff
@@ -56,10 +56,9 @@ def import_ti_core(tmp_dir=None):
 def locale_encode(s):
     try:
         import locale
-        encoding = locale.getdefaultlocale()[1]
+        return s.encode(locale.getdefaultlocale()[1])
```

Comment:
Just FYI in some **rare** Linux setups, such as the Ubuntu docker base image, `locale.getdefaultlocale()[1]` will return `None` thus `s.encode` will fail, which is why:
https://github.com/taichi-dev/taichi/pull/1385/files#diff-3254677a7917c6c01f55212f86c57fbfR57 
is needed.

Reply:
Yes, so I changed this to make `LANG=C` work.

Reply:
👍  Would be even more clear to catch the specific error (TypeError)